src/main/java/creativei/enums/CaseIndex.java

```diff
@@ -13,47 +13,66 @@
 @JsonFormat(shape = JsonFormat.Shape.OBJECT)
 public enum CaseIndex {
 
-    UNLIKELY("Unlikely",1),
-    INTERESTED("Intersted",2),
-    LIKELY("Likely",3),
-    HOT_LEAD("Hot Lead",4);
+    UNLIKELY("Unlikely", 1),
+    INTERESTED("Intersted", 2),
+    LIKELY("Likely", 3),
+    HOT_LEAD("Hot Lead", 4);
 
     private static final Logger logger = LoggerFactory.getLogger(CaseIndex.class);
-    private static ObjectMapper mapper=new ObjectMapper();
+    private static ObjectMapper mapper = new ObjectMapper();
```

Comment:
Instead of creating object here it would be better to create the object and reference inside static block to allow garbage collection on it.

Reply:
resolved in commit 31c4170